startup/95-plans.py

```diff
@@ -120,29 +120,11 @@ def inner():
         yield from bps.abs_set(mono1.start_trajectory, '1', wait=True)
 
         # this should be replaced by a status object
+        # Updated below method. Should we remove the comment above?
         def poll_the_traj_plan():
-            while True:
-                ret = (yield from bps.read(mono1.trajectory_running))
-                if ret is None:
-                    break
-                is_running = ret['mono1_trajectory_running']['value']
-
-                if is_running:
-                    break
-                else:
-                    yield from bps.sleep(.1)
-
-            while True:
-                ret = (yield from bps.read(mono1.trajectory_running))
-                if ret is None:
-                    break
-                is_running = ret['mono1_trajectory_running']['value']
-
-                if is_running:
-                    yield from bps.sleep(.05)
-                else:
-                    break
 
+            yield from bps.mv(mono1, 'start')
+            yield from bps.sleep(delay)
```

Comment:
I think we don't have the `delay` kwarg in the function. Please remove the line above.

Reply:
From both "prepare" and here in "start". I believe "prepare" also have the same line

Reply:
'prepare' function has it, but not this 'start' one. Olease check the definition of the function and the signature.

Reply:
Got it.